HHS/Head-Start-TTADP
# [Prod] Fixed FEI goals being closed and new FEI goals created for multiple recipient ARs, apply UI to fix to make submitting report easier

## Description of change

This change includes the UI for editing FEI goals on multi-recipient reports. 

![Screenshot 2024-03-15 at 11 25 26 AM](https://github.com/HHS/Head-Start-TTADP/assets/3288586/1d9d7d97-8f6f-4aa1-a50d-1465d243e4fc)

It also includes a fix that should stop extra goals from being created arbitrarily, updates assorted software dependencies, and deletes a communication log per user request
 
## How to test

The user's specific case was tested on prod like data. To see the new UI in action, [review this report](https://tta-smarthub-staging.app.cloud.gov/activity-reports/36788/review). The alert at the bottom has an arrow that can be toggled (see screenshot).

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2717


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
